scripts/datasets/light_microscopy/check_omnipose.py

```diff
@@ -0,0 +1,21 @@
+from torch_em.util.debug import check_loader
+from torch_em.data import MinInstanceSampler
+from torch_em.data.datasets.light_microscopy import get_omnipose_loader
+
+
+ROOT = "/media/anwai/ANWAI/data/omnipose"
```

Comment:
Adapt to the new ROOT logic. See also comments in https://github.com/constantinpape/torch-em/pull/330